client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -114,7 +113,7 @@ export default React.createClass( {
 			ref: name,
 			additionalClasses: 'checkout-field',
 			value: formState.getFieldValue( this.state.form, name ),
-			invalid: formState.isFieldInvalid( this.state.form, name ),
+			isError: formState.isFieldInvalid( this.state.form, name ),
```

Comment:
Yes, they are expecting `isError` too.
I selected `isError` here, because that's what we use in the underlying `form-text-input`. I've added the same prop to `FormSelect`, which is backing `StateSelect` and `CountrySelect`. So everything here should be "isError-ready".